e2e_tests/tests/command/test_tensorboard.py

```diff
@@ -53,6 +53,7 @@ def s3_config(num_trials: int, secrets: Dict[str, str]) -> str:
 """
 
 
+@pytest.mark.skip()  # type: ignore
```

Comment:
This are skipped because, as far as I understand, we are pretending the /tmp folder is a shared file system, but now that the Tensorboard is running on a different instance than the experiment, that doesn't hold true so the tests can't pass.

Reply:
test_pytorch_load assumes this as well - maybe we should have the test masters spin up a little NFS server to replace our use of /tmp or something.

Reply:
blocking: it shouldn't matter if shared_fs actually uses shared fs. The TensorBoard will access the /tmp folder, where stores the checkpoints. I don't really understand the reasoning 

> "but now that the Tensorboard is running on a different instance than the experiment, that doesn't hold true so the tests can't pass." 

TensorBoard has always been running on a different instance. What has been changed?

Reply:
If using a different resource pool, TensorBoard will be running on a different agent than the experiment. But this is an e2e cpu test, all the tasks should run on the same agent

Reply:
And it shouldn't matter if TensorBoard actually uses a shared fs or just a folder.

Reply:
This is not an e2e CPU test , it is run as part of the e2e GPU test - https://app.circleci.com/pipelines/github/determined-ai/determined/9229/workflows/a07e5abd-b93e-4cf0-9849-f5860f519ead/jobs/255468

LMK if I have root caused this incorrectly, but based on the fact that it works if we run the tensorboard in the GPU pool, I believe this is the cause of the 'metrics not found'

Reply:
The link you posted is `test_start_tensorboard_for_multi_experiment`. We should change it to be `e2e_cpu` instead.

Reply:
Oh you're right , I'll remove this skip